packages/graphback-codegen-resolvers/src/templates/LayeredResolverTemplates.ts

```diff
@@ -2,43 +2,43 @@
 // TODO support for the database type instead of hardcoded lowercase
 export const createTemplate = (fieldName: string, typeName: string, subscription?: boolean): string => {
   return `${fieldName}: (_, args, context) => {
-      validateContext(context)
+      validateRuntimeContext(context)
       return context.crudService.create("${typeName.toLowerCase()}", args.input, context);
     }`
 }
 
 export const updateTemplate = (fieldName: string, typeName: string, subscription?: boolean): string => {
   return `${fieldName}: (_, args, context) => {
-      validateContext(context)
+      validateRuntimeContext(context)
       return context.crudService.update("${typeName.toLowerCase()}", args.id, args.input, context);
     }`
 }
 
 export const deleteTemplate = (fieldName: string, typeName: string, subscription?: boolean): string => {
   return `${fieldName}: (_, args, context) => {
-      validateContext(context)
+      validateRuntimeContext(context)
       return context.crudService.delete("${typeName.toLowerCase()}", args.id, args.input, context);
     }`
 }
 
 export const findAllTemplate = (fieldName: string, typeName: string): string => {
   return `${fieldName}: (_, args, context) => {
-      validateContext(context)
+      validateRuntimeContext(context)
       return context.crudService.findAll("${typeName.toLowerCase()}", context);
     }`
 }
 
 export const findTemplate = (fieldName: string, typeName: string, ): string => {
   return `${fieldName}: (_, args, context) => {
-      validateContext(context)
+      validateRuntimeContext(context)
       return context.crudService.findBy("${typeName.toLowerCase()}", args.fields, context);
     }`
 }
 
 export const newSub = (typeName: string): string => {
   return `new${typeName}: {
       subscribe: (_: any, __: any, context: GraphQLContext) => {
```

Comment:
Should `GraphQLContext` be `GraphbackRuntimeContext`?

GraphQLContext does not have the `crudDb` or `crudService` properties.